ami/models/components/audio_patchifier.py

```diff
@@ -0,0 +1,81 @@
+# Ref:
+# https://github.com/facebookresearch/fairseq/blob/main/fairseq/models/wav2vec/wav2vec2.py#L844
+# https://github.com/facebookresearch/fairseq/blob/main/examples/data2vec/models/modalities/audio.py#L29
+
+import dataclasses
+
+import torch
+import torch.nn as nn
+
+
+class ConvBlock(nn.Module):
+    def __init__(
+        self,
+        in_channels: int,
+        out_channels: int,
+        kernel_size: int,
+        stride: int,
+    ) -> None:
+        super().__init__()
+
+        self.conv = nn.Conv1d(in_channels, out_channels, kernel_size=kernel_size, stride=stride, bias=False)
+        nn.init.kaiming_normal_(self.conv.weight)
+        self.layer_norm = nn.LayerNorm(out_channels, elementwise_affine=True)
+        self.non_linear_layer = nn.GELU()
```

Comment:
`non_linear_layer`という名称は不明瞭。
```suggestion
        self.activation= nn.GELU()
```